css/main.css

```diff
@@ -413,8 +419,10 @@ h1{
 	margin-left: 0.5em; /* To put space between scroll bar of main panel and this bar to avoid accidental scrolling. */
 }
 .mainPanel{
+	margin: 0 0 0 0.5em; /* Main panel only has a margin on the left so that iframe scroll bars aren't displaced. */
+	padding: 0.5em 0 0.5em 0; /* Content padding so we don't get that background color clipping over the top/bottom of content */
 	min-width: 10%;
-	overflow-y: scroll; /* Changing this to auto crashes things :/ */
+	overflow-y: auto; /* Changing this to auto crashes things :/ */
```

Comment:
if we don't crash anything now, do we need the comment?

Reply:
true, I was so happy it finally worked that I forgot to clean house.